libpod/networking_linux.go

```diff
@@ -121,6 +121,19 @@ func (r *Runtime) createNetNS(ctr *Container) (n ns.NetNS, q []*cnitypes.Result,
 	return ctrNS, networkStatus, err
 }
 
+type slirp4netnsCmdArg struct {
+	Proto     string `json:"proto,omitempty"`
+	HostAddr  string `json:"host_addr"`
+	HostPort  int16  `json:"host_port"`
```

Comment:
uint16?

Reply:
I restored `int32` that is the same type we are using PortMapping